pkg/network/multus/nad_test.go

```diff
@@ -48,21 +48,21 @@ var _ = Describe("NetworkAttachmentDefinition", func() {
 		})
 	})
 
-	Context("GetResourceNameForNetwork", func() {
+	Context("ResourceNameFromNetAttachDef", func() {
```

Comment:
Shouldn't it test `GetNetworkToResourceMap` instead to cover the entire logic?

Reply:
It would be much better to test the entire logic and not testing this implementation detail.
As I wrote in the PR description, this is just a preliminary step that will allow us to refactor this logic further and add test additional test coverage in a follow-up.

Reply:
That's fair, thanks for clarifying.